subprojects/launcher/src/main/java/org/gradle/tooling/internal/provider/FileSystemWatchingBuildActionRunner.java

```diff
@@ -78,6 +78,7 @@ public Result run(BuildAction action, BuildTreeLifecycleController buildControll
         }
 
         boolean actuallyWatching = virtualFileSystem.afterBuildStarted(watchFileSystemMode, verboseVfsLogging, debugWatchLogging, buildOperationRunner);
+        LOGGER.info("File system watching is {}", actuallyWatching ? "active" : "inactive");
```

Comment:
I wonder if we could merge the two info log messages into one, something like
```
Gradle is watching the file system for changes
Gradle tried to watch the file system for changes but couldn't
Gradle is not watching the file system for changes
```

Reply:
I think for `INFO` logs human readability is not that important. Here I actually prefer the two separate log messages, as if something bad happens before the second one could be printed, we still print the first one.

Reply:
Fine with me.